packages/macro/tsconfig.json

```diff
@@ -1,7 +1,10 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "lib": [
+      "ES2019.Array"
+    ],
```

Comment:
This is for Array.prototype.flat / Array.prototype.flatMap 
Supported started from Node.js 11 AFAIK